stubs/pyOpenSSL/OpenSSL/SSL.pyi

```diff
@@ -115,25 +117,63 @@ def SSLeay_version(type: int) -> str: ...
 class Session: ...
 
 class Connection:
-    def __getattr__(self, name: str) -> Any: ...  # incomplete
+    def __getattr__(self, name: str) -> Any: ...  # takes attributes from `self._socket`
     def __init__(self, context: Context, socket: socket.socket | None = ...) -> None: ...
+    def get_context(self) -> Context: ...
+    def set_context(self, context: Context) -> None: ...
+    def get_servername(self) -> bytes | None: ...
+    def set_tlsext_host_name(self, name: bytes) -> None: ...
+    def pending(self) -> int: ...
+    def send(self, buf: ReadableBuffer, flags: int = ...) -> int: ...
```

Comment:
This also accepts `str` (though it's deprecated): https://github.com/pyca/pyopenssl/blob/51717970261d6283b681f5fa8347808409f0991d/src/OpenSSL/SSL.py#L1708

```suggestion
    def send(self, buf: ReadableBuffer | str, flags: int = ...) -> int: ...
```